src/Model/Entity.php

```diff
@@ -116,7 +116,7 @@ public function setODataType($val)
     *
     * @return array The list of properties
     */
-    public function jsonSerialize()
+    public function jsonSerialize(): array
```

Comment:
I guess the intent was to avoid the deprecation notice introduced in PHP 8.1 (and a future error in PHP 9), but sorry, that's a [non-documented] **breaking change** in a *non-major* release! Please compare:
- with 1.65.0: https://3v4l.org/TgIRr
- with 1.66.0: https://3v4l.org/LuoZu

To support both PHP 7.3+ and 8.0+, [I think] that change should be reverted, and do this instead:

```diff
+    #[ReturnTypeWillChange]
     public function jsonSerialize()
```

@Ndiritu WDYT?